packages/x-date-pickers/src/locales/utils/pickersLocaleTextApi.ts

```diff
@@ -70,8 +70,28 @@ export interface PickersComponentAgnosticLocaleText<TDate extends PickerValidDat
   selectViewText: (view: TimeViewWithMeridiem) => string;
 
   // Open picker labels
-  openDatePickerDialogue: (date: TDate | null, utils: MuiPickersAdapter<TDate>) => string;
-  openTimePickerDialogue: (date: TDate | null, utils: MuiPickersAdapter<TDate>) => string;
+  openDatePickerDialogue: (
+    /**
+     * @deprecated Use `formattedTime` instead
+     */
+    date: TDate | null,
+    /**
+     * @deprecated Use `formattedTime` instead
+     */
+    utils: MuiPickersAdapter<TDate>,
+    formattedDate: string | null,
+  ) => string;
+  openTimePickerDialogue: (
+    /**
+     * @deprecated Use `formattedTime` instead
+     */
+    date: TDate | null,
+    /**
+     * @deprecated Use `formattedTime` instead
+     */
+    utils: MuiPickersAdapter<TDate>,
+    formattedTime: string | null,
+  ) => string;
```

Comment:
I believe that `clockLabelText` just above on line 64 also uses utils but its arg is named `adapter`, should it get the same treatment as these two?

Reply:
Nice catch!
I'll do the same there :+1: